RELEASE.md

```diff
@@ -10,6 +10,42 @@ ______________________________________________________________________
 
 ### Bug fixes
 
+
+## Release v0.4.2 - June 8. 2022
+
+### Incident postmortem
+
+#### Bitnami update breaks post v0.4.0 releases
+
+On June 2, 2022, GitHub user @peytondmurray reported [issue 1306](https://github.com/Quansight/qhub/issues/1306), stating that he was unable to deploy QHub using either the latest release `v0.4.1` or installing `qhub` from `main`. As verified by @peytondmurray and others, during you first `qhub deploy`, the deployment halts and complains about two invalid Helm charts missing from the bitnami `index.yaml`.
+
+[Bitnami's decision to update how long they keep old Helm charts in their index for](https://github.com/bitnami/charts/issues/10539) has essentially broken all post `v0.4.0` versions of QHub.
+
+This is a sever bug that will affect any new user who tries to install and deploy QHub with any version less then `v0.4.2` or greater than or equal to `v0.4.0`.
+
+Given the impact and severity of this bug, the team has decided to quickly cut a hotfix.
+
+#### AWS deployment failing due to old auto-scaler helm chart
+
+On May 27, 2022, GitHub user @tylerpotts reported [issue 1302](https://github.com/Quansight/qhub/issues/1302), stating that he was unable to deploy QHub using the latest release `v0.4.1` (or installing `qhub` from `main`). As described in the original issue, the deployment fails complaining about the deprecated `v1beta` Kubernetes API. This lead to the discovery that we were using an outdated `cluster_autoscaler` helm chart.
+
+The solution is update from `v1beta` to `v1` Kubernetes API for the appropriate resources and update the reference to the `cluster_autoscaler` helm chart.
+
+Given the impact and severity of this bug, the team has decided to quickly cut a hotfix.
+
+### Bug fixes
+
+This release is a hotfix for the issue summarized in the following:
+* [issue 1319](https://github.com/Quansight/qhub/issues/1319)
+* [issue 1306](https://github.com/Quansight/qhub/issues/1306)
+* [issue 1302](https://github.com/Quansight/qhub/issues/1302)
+
+
+### What's Changed
```

Comment:
Hi @iameskild is this hotfix a cherry-pick only for the above-mentioned issues, or does it also will include the changes made to `main` since the previous release? just to confirm as [diff](https://github.com/Quansight/qhub/compare/v0.4.1...v0.4.2rc2) points to the first.